backend/config/rewards_schedule.yml

```diff
@@ -0,0 +1,20 @@
+-
+  prize: $20 Starbucks gift card
+  points: 75
+  start_at: 2022-01-18
+  end_at: 2022-02-14
+-
+  prize: $50 Amazon gift card
+  points: 100
+  start_at: 2022-02-15
+  end_at: 2022-03-14
+-
+  prize: $50 Nike gift card
+  points: 50
+  start_at: 2022-03-15
+  end_at: 2022-04-14
+-
+  prize: Apple AirPods™
+  points: 0
+  start_at: 2022-04-20
+  end_at: 2022-04-20
```

Comment:
This is read on startup, so can (and should) be overridden by the deployment, right?

Reply:
no, this is the "real" schedule.  eventually we'll store it in the DB and write an admin UI to allow modifications